api/core/storage/mongo/migrations/migrations/m2025_05_01_tools_indices.py

```diff
@@ -0,0 +1,34 @@
+from core.storage.mongo.migrations.base import AbstractMigration
+
+
+class AddToolsIndicesMigration(AbstractMigration):
+    """Add necessary indices to the tools collection"""
+
+    @property
+    def _tools_collection(self):
+        return self.storage._tools_collection  # pyright: ignore [reportPrivateUsage]
+
+    async def apply(self):
+        # Index on tenant for efficient queries by tenant
+        await self._tools_collection.create_index(
```

Comment:
no need to create that index. Compound indices work when querying on prefix only so efficient queries will be covered by the `tools_tenant_name_unique` index

Reply:
thanks https://github.com/WorkflowAI/WorkflowAI/pull/204/commits/5948382c8e7debf71e4908627d694a4ebe81cc6d